examples/lstm_text_generation.py

```diff
@@ -30,8 +41,8 @@
 indices_char = dict((i, c) for i, c in enumerate(chars))
 
 # cut the text in semi-redundant sequences of maxlen characters
-maxlen = 40
-step = 3
+maxlen = param['maxlen']
+step = param['step']
```

Comment:
Please do not pack the values in a dict, rather, defined named variables here, to be used later in the code.